Unable to compile TypeScript following "Get Started" Guide - Could not find a declaration file for module '@prisma/client'

**Set up:**
- `@prisma/client: 2.0.0-preview021`
- `Prisma2: 2.0.0-preview021`

While following the newest [Get Started Guide](https://github.com/prisma/prisma2/blob/master/docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md) upon reaching the last step and trying to execute the `index.ts` script, I got the following error:

```
src/index.ts:1:30 - error TS7016: Could not find a declaration file for module '@prisma/client'. '/Users/robinmacpherson/robinmacpherson/backend/node_modules/@prisma/client/index.js' implicitly has an 'any' type.
  Try `npm install @types/prisma__client` if it exists or add a new declaration (.d.ts) file containing `declare module '@prisma/client';`

1 import { PrismaClient } from '@prisma/client'
```

**Screenshot:**
![image](https://user-images.githubusercontent.com/34203886/74455925-ed07c300-4e3a-11ea-9200-75e35018113b.png)

**src/index.ts**
```
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

// A `main` function so that we can use async/await
async function main() {
  const user1 = await prisma.user.create({
    data: {
      email: 'alice@prisma.io',
      name: 'Alice',
      posts: {
        create: {
          title: 'Watch the talks from Prisma Day 2019',
          content: 'https://www.prisma.io/blog/z11sg6ipb3i1/',
          published: true,
        },
      },
    },
    include: {
      posts: true,
    },
  })
  const user2 = await prisma.user.create({
    data: {
      email: 'ryan@example.com',
      name: 'Ryan',
      posts: {
        create: [
          {
            title: 'Robin Is The Best!',
            content: "I'm telling you, it's true!",
            published: true,
          },
          {
            title: 'Journaly Is Going To Be Great',
            content: 'I am so excited to test out Journaly',
            published: false,
          },
        ],
      },
    },
    include: {
      posts: true,
    },
  })

  console.log(
    `Created users: ${user1.name} with (${user1.posts.length} post) and ${user2.name} with (${user2.posts.length} posts) `,
  )
}

main()
  .catch(e => console.error(e))
  .finally(async () => {
    await prisma.disconnect()
  })

```

--- a/docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md
+++ b/docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md
@@ -69,6 +69,11 @@ Follow these steps for an initial Prisma setup:
     npx prisma2 migrate save --name 'init' --experimental
     npx prisma2 migrate up --experimental
     ```
+1. Generate the Prisma Client with your updated datamodel::
+   ```
+   npx prisma2 generate --experimental
+   ```
+   _Please note that this step is unnecessary at this stage if you installed the `@prisma/client` dependency (step 7) after creating your datamodel (steps 4 - 6), as our post-install hook will automatically generate the Prisma Client for you based on the existing data model. However, it is helpful to be aware of `prisma2 generate` as you will need to run this command any time that you update your data model while evolving your app._
 1. Run `touch index.ts` to create a source file and add the following code:
     ```ts
     import { PrismaClient } from '@prisma/client'

--- a/docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md
+++ b/docs/getting-started/start-from-scratch-with-empty-db/quickstart-prisma-migrate.md
@@ -71,7 +71,7 @@ Follow these steps for an initial Prisma setup:
     ```
 1. Generate the Prisma Client with your updated datamodel::
    ```
-   npx prisma2 generate --experimental
+   npx prisma2 generate
    ```
    _Please note that this step is unnecessary at this stage if you installed the `@prisma/client` dependency (step 7) after creating your datamodel (steps 4 - 6), as our post-install hook will automatically generate the Prisma Client for you based on the existing data model. However, it is helpful to be aware of `prisma2 generate` as you will need to run this command any time that you update your data model while evolving your app._
 1. Run `touch index.ts` to create a source file and add the following code:

